### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for document in "Deploy Web App dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1395]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app.
4. "Deploy web app" dialog will open. 
5. In "Deploy web app" dialog, observe the name property for the document using Inspect.

**Actual Result:**
Name property is empty for the document..
Narrator is not announcing the value.value information for the document.

**Narrator behaviour:**
Tab

**NVDA behaviour:**
edit  read only
blank

**Expected Result:**
Name property for the document should be "Deploy Web App".
Narrator and NVDA should announce as "Deploy Web App Select App Service to deploy to"

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![deploy web app-document](https://user-images.githubusercontent.com/22956130/35683652-ab06d020-078a-11e8-8003-01975719680a.jpg)
![deploy web app-document-nvda](https://user-images.githubusercontent.com/22956130/35683661-adf075ca-078a-11e8-8c4a-ddfe8ef73b79.jpg)

**Comment by Eskibear** [comment]
ref #1407

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can still see the property in the `Value` field
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.


#### Workaround
Users can still see the property in the `Value` field

![image](https://user-images.githubusercontent.com/2351748/46598104-5f64e680-cb15-11e8-874c-8d1c6419f5d8.png)

**Comment by jdneo** [comment]
Can show the name property on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0

![image](https://user-images.githubusercontent.com/6193897/47150232-9357bd00-d308-11e8-8a4d-f79e655e4920.png)

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749842
